spec/models/manageiq/providers/openshift/container_manager/refresher_spec.rb

```diff
@@ -116,15 +116,15 @@ def full_refresh_test
     let(:object_counts) do
       # using strings instead of actual model classes for compact rspec diffs
       {
-        'ContainerProject'           => 10,
-        'ContainerImage'             => 39,
+        'ContainerProject'           => 18,
+        'ContainerImage'             => 67,
```

Comment:
I see only 57 images in the .txt files.  Let me check the yml...

Reply:
57 in yml too.
```
$ cat spec/vcr_cassettes/manageiq/providers/openshift/container_manager/refresher_before_deletions.yml | yq '.http_interactions[] | select(.request.uri | test(".*/images")) | .response.body.string' --raw-output |  jq '.items[] | .dockerImageReference' | tee images_before_deletions | wc
     57      57    6336
 cat spec/vcr_cassettes/manageiq/providers/openshift/container_manager/refresher_after_deletions.yml | yq '.http_interactions[] | select(.request.uri | test(".*/images")) | .response.body.string' --raw-output |  jq '.items[] | .dockerImageReference' | tee images_after_deletions | wc
     57      57    6336
$ diff images_before_deletions images_after_deletions -s                                                          2019-09-20T16:18:31+03:00
Files images_before_deletions and images_after_deletions are identical
```

Reply:
There were 34 before:
```
grep 'sha256' spec/vcr_cassettes/manageiq/providers/openshift/container_manager/refresher_before_deletions.txt  | wc -l
34
```

And we were expecting 39.  I don't think it is incorrect for these to be different as we build container images for containers which aren't in the registries.

Reply:
Ah, right there are also images from pods.
As explained here:
```ruby
  let(:all_images_count) { 40 } # including /oapi/v1/images data
  let(:pod_images_count) { 12 } # only images mentioned by pods
  let(:images_managed_by_openshift_count) { 32 } # only images from /oapi/v1/images
```
These numbers didn't need updating because they're for tests using `refresher.yml` which you didn't re-record.  Indeed:
```sh
$ cat spec/vcr_cassettes/manageiq/providers/openshift/container_manager/refresher.yml | yq '.http_interactions[] | select(.request.uri | test(".*/images")) | .response.body.string' --raw-output |  jq '.items[] | .dockerImageReference' | tee images | wc
     32      32    3275
```

Reply:
:+1:

Reply:
LGTM :100: